### DISCUSSION TITLE
Incomplete Data

### DISCUSSION THREAD

**Discussion by HafizRiaz** [community/community#57735] [Discussions]
### Select Topic Area

Bug

### Body

I found a paper from science direct in which it was claimed that the, "For reproduction " we have made our code in case study open source. When i clicked at the link of GitHub, just python files were available but excel data file was missing.  Due to this missing file code i nor working, rather asking for that specific file. instead of requesting, owner did not respond. what should i do now? please help. The link for that reference is given below. Corresponding paper is in attachment 
https://github.com/zsh1109/IJEPES-code/blob/e69db3ee8d30d2f1afbe3c8748fcf42c3d2b2528/code.zip. 
[MADRL.pdf](https://github.com/community/community/files/11714349/MADRL.pdf)

**Comment by airtower-luna** [comment]
That's probably something best discussed with the journal or the author's academic institution. Us other users can't really help with what they did and didn't publish. :thinking:

**Comment by HafizRiaz** [comment]
Thanks for the response. Actually I already contacted the corresponding author but he did not responding. As I am very new to this forum as well as to python, searching for help.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by airtower-luna** [comment]
You mentioned that you already tried to contact the *author*, yes. What I meant was to contact the *journal* or the *academic institution* the author is part of (e.g. if they're a PhD student, their institute office). Both should have an interest in making sure published papers & data are sound, and could prod the author about it.

**Reply by airtower-luna** [reply]
Important addition: For the author's sake, make sure not to use an accusatory tone if you contact the journal or their institution. Just say you can't find the data, and if they could provide it or ask the author to. :wink: 

If you have trouble with Python here's a good place to ask, but that should probably go into the Programming Help category, with the specific questions you have. :slightly_smiling_face:

**Comment by HafizRiaz** [comment]
Thanks for the suggestion. I really appreciate your way of guidence.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
